<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="3">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Space</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxSpace">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Param</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolButtonRefresh">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="Experiment.qrc">
       <normaloff>:/openicon/icons/view-refresh-7.png</normaloff>:/openicon/icons/view-refresh-7.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>From</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QDateTimeEdit" name="dateTimeEditFrom"/>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxParam">
     <property name="maxVisibleItems">
      <number>1000</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="loadButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="Experiment.qrc">
       <normaloff>:/openicon/icons/document-revert-5.png</normaloff>:/openicon/icons/document-revert-5.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="namedTimespanComboBox">
     <property name="maxVisibleItems">
      <number>1000</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Experiment.qrc"/>
 </resources>
 <connections/>
</ui>
